lib/redcord/migration/index.rb

```diff
@@ -48,7 +48,7 @@ def _del_set(model, key)
   sig { params(model: T.class_of(Redcord::Base), key: String).void }
   def _del_zset(model, key)
     # ZPOPMIN might not be avaliable on old redis servers
-    model.redis.zscan_each(match: key) do |id, _|
+    model.redis.zscan_each(nil, match: key) do |id, _|
```

Comment:
After digging into this a bit more. I'm pretty sure this is a bug

I think what this code to remove an index is trying to do is
- iterate through the set which contains all the index attributes
- delete each item one by one
- then just delete the set all together

However we could just delete the set itself, and that will empty the contents of the set. I also dug up this comment, I think we can just replace deleting the set with unlink: https://github.com/chanzuckerberg/redcord/pull/72/files#r504317175

```suggestion
    model.redis.zscan_each(key) do |id, _|
```
For now, I think this is what it should be. I'll create an issue separately to use unlink instead of this existing logic

Reply:
https://github.com/chanzuckerberg/redcord/issues/113

Reply:
Gotcha, that seems simple enough - I'll do that in a separate issue?